### PR TITLE
add minimum tolerance for number input dragging (fixes #42)

### LINKED ISSUE #42: Clicking on inputs can trigger accidentally rewrite values (due to horizontal stepping)
I just clicked on the field for the rotation X value, and it changed to `-0.02` because I must've dragged my mouse a bit on `mousedown`.

image

### PR DESCRIPTION
**Pull request by ngokevin**
Change at: https://github.com/aframevr/aframe-editor/compare/master...ngokevin:numtol?expand=1#diff-dc291811480fae652eda7cd59e52b139R47

### CODE REVIEW

File: src/components/widgets/NumberWidget.js [hunk 1]
```diff
@@ -120,12 +130,12 @@ export default class NumberWidget extends React.Component {
   render () {
     return (
       <input ref='input' className='number' type='text'
-        value={this.state.displayValue}
-        onKeyDown={this.onKeyDown}
-        onChange={this.onChange}
-        onMouseDown={this.onMouseDown}
-        onFocus={this.onFocus}
-        onBlur={this.onBlur}
+             value={this.state.displayValue}
```
Comment (dmarcos) [inline src/components/widgets/NumberWidget.js]: Is this indentation correct?

Comment (ngokevin) [inline src/components/widgets/NumberWidget.js]: The default React linter doesn't have indent rules enabled. I've usually indented to line up the props. But by 2s is accepted as well.

Comment (dmarcos) [inline src/components/widgets/NumberWidget.js]: What do you mean accepted as well? There should be just one way to indent. Just pick one.

Comment (ngokevin) [inline src/components/widgets/NumberWidget.js]: i'll pick to line up visually with the first prop then

Comment (dmarcos) [inline src/components/widgets/NumberWidget.js]: No. it's not a criteria you can consistently apply everywhere. Just use two spaces. You start to have context dependent indentation if you go that route. There's zero value on it.

Comment (dmarcos) [inline src/components/widgets/NumberWidget.js]: It's @fernandojsg code base so I let him make the call

Comment (fernandojsg) [inline src/components/widgets/NumberWidget.js]: I don't really have an strong opinion about the indent, but I would prefer to be consistent everywhere and not context dependent yep.
@ngokevin can we enable some indent rules in the linter even if by default they're disabled?

Comment (ngokevin) [inline src/components/widgets/NumberWidget.js]: yeah

Comment (fernandojsg) [inline src/components/widgets/NumberWidget.js]: Perfect, thanks!

**Pull request merged**